sss/tests/test_sss.py

```diff
@@ -29,9 +29,14 @@ def test_check_extra_columns(setup_and_teardown_package):
 
     columns_to_check = ['miscellaneous_is_secondary', 'health_care_is_secondary', 'analysis_is_secondary']
 
-    for col in columns_to_check:
-        assert col in df.columns
-
```

Comment:
why take out this check?

Reply:
I wanted to test whether that specific column is empty or not empty. But I think both methods works.